Jenkinsfile

```diff
@@ -1,4 +1,6 @@
 /*
 * `buildPlugin` step provided by: https://github.com/jenkins-infra/pipeline-library
 */
-buildPlugin(platforms: ['linux'], jdkVersions: ['11'])
+buildPlugin(useContainerAgent: true, configurations: [
+        [ platform: "linux", jdk: "11" ]
```

Comment:
You may also consider testing on 17 as in https://github.com/jenkinsci/archetypes/pull/547 etc.

Reply:
I want to, and tried but tests are failing on 17, have to figure out why

Reply:
What specifically failed? We expect all plugin builds to pass on 17 these days.